### ISSUE TITLE
Berkshelf throw error when uploading memcached 1.9.0

### ISSUE THREAD

**Issue by mikaelhm** [chef-cookbooks/memcached#55]
Hi, 

when uploading memcached 1.9.0 with berkshelf to my chef-server, it get this error:

```
E, [2015-11-09T11:41:25.999906 #39632] ERROR -- : Ridley::Errors::HTTPBadRequest: {"error":["Invalid element in array value of 'templates'."]}
```

1.8.0 works without issues.

**Comment by tas50** [comment]
Can you test this with the 2.0 release

**Comment by mikaelhm** [comment]
I get the same error in 2.0.
However, I managed to track it down;

I use Berkshelf to install memcahced 2.0.0 on my mac.
That creates a folder at  `~/.berkshelf/cookbooks/memcached-2.0.0` with the cookbook.

However in  `~/.berkshelf/cookbooks/memcached-2.0.0/templates/` I a `.DS_Store` file.
This is the file that chef-server does not like.

Looks like this file might have been created when you packed it up for a release on berkshelf:
https://github.com/berkshelf/berkshelf/issues/706

**Comment by tas50** [comment]
I just pushed a new artifact with stove (2.0.1).  There should't be any sort of ds_store mess in there.  Give it a try if you can please.

**Comment by mikaelhm** [comment]
I still have the issue in 2.0.1

/m

**Comment by tas50** [comment]
I'm really confused how this file was introduced since stove should wipe it before upload, but I've removed all ds_store files from my repos and pushed 2.0.2.  Open this up if you still have issues.

**Comment by mikaelhm** [comment]
I just verified this is fixed in 2.0.2.

**Comment by gerard-leijdekkers** [comment]
Same problem in 3.0.1. No `.DS_Store` file present. I do not know how to find the culprit.
